assignment1.md

```diff
@@ -0,0 +1,2 @@
+![](images/MajorTomDiff.PNG)
```

Comment:
Maybe better to add description or explanation of why there're irregular peaks in difference for `MajorTom16.wav`